composer.json

```diff
@@ -29,7 +29,8 @@
         "symfony/process": "~2.4"
     },
     "require-dev": {
-        "psr/log": "~1.0"
+        "psr/log": "~1.0",
+        "phpunit/phpunit": "~4.0"
```

Comment:
although, because the version constraint in it's composer file is incorrect, it does let you install it on php 7